po/base/zh_CN.po

```diff
@@ -574,7 +574,7 @@ msgstr "引导期间"
 #: library/cwm/src/modules/CWMServiceStart.rb:266
 #: library/cwm/src/modules/CWMServiceStart.rb:321
 msgid "Manually"
-msgstr "手动"
+msgstr "手动 (&M)"
```

Comment:
The translator comment explicitly says **NO SHORTCUT**; this is part of a help text where that shortcut would apear literally. Please remove.

Reply:
Adding the shortcut causes it to be displayed literally in the help window:

![2018-11-15 11-56-37](https://user-images.githubusercontent.com/1259034/48548721-f586ec00-e8cd-11e8-9a38-95f2f7f86421.png)

Removing it fixes the help, but the shortcut is not created any more:

![2018-11-15 11-58-01](https://user-images.githubusercontent.com/1259034/48548724-f6b81900-e8cd-11e8-992e-6d4a54af784a.png)

@shundhammer, Is this a bug in YaST core? If a shortcut should be automatically created for "Manually" and "During boot", is this another bug to be handled separately?

From my point of view, I would prefer to have a working shortcut even if it is displayed literally in the help window rather than correct help strings without a working shortcut, at least until the bug in YaST core is fixed.